Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8
```

Comment:
Maybe we can go to the last stable 3.9

Reply:
will do

Reply:
@XaviTorello djangorestframework-simplejwt==4.4.0 is not a valid version for python 3.9. The most recent available for this version is 4.3.0. Any thoughts?

Reply:
I am starting to get lots of errors with the application once upgraded to 3.9..

Reply:
I tried downgrade the pip package and although we can install all other requirements I started to get other kind of issues from the `base64` module. May I suggest to handle the upgrade to python 3.9 as a separate PR? 
I personally think it's more manageable to work with smaller scopes.

Reply:
makes sense, maybe it's better to wait some days until the community upgrade their libs 💪 

many thanks @mcastellin